github/messages.go

```diff
@@ -30,6 +31,35 @@ const (
 	sha512Prefix = "sha512"
 	// signatureHeader is the GitHub header key used to pass the HMAC hexdigest.
 	signatureHeader = "X-Hub-Signature"
+	// eventTypeHeader is the Github header key used to pass the event type.
+	eventTypeHeader = "X-Github-Event"
+)
+
+var (
+	// eventTypeMapping is a conversion map between go-github types and webhooks types.
```

Comment:
What about improving the phrasing to say:

```
// eventTypeMapping maps webhooks types to their corresponding go-github struct types.
```

Or is that not a more accurate statement?